user_scripts/variant_calling.sh

```diff
@@ -43,6 +43,7 @@ singularity_bindings="-B $project_resources:/opt/project_resources
  -B $species_resources:/opt/species_resources
  -B $wrangler_directory:/opt/data
  -B $output_directory:/opt/analysis
+ -B /home/charlie/projects/MIPTools_wrangler_in_sif/snakemake:/opt/snakemake
```

Comment:
This location probably shouldn't be hard coded. Maybe it gets changed in a later commit?